### ISSUE TITLE
Improve Appearance of the Chart over Longer Time Intervals

### ISSUE THREAD

**Issue by UPstartDeveloper** [C3NZ/visualizing-diet#3]
Currently, the project uses a 4:3 aspect ratio. Possibly there is a way that I can adjust this so the dates on the x-axis are not so bunched up together?

**Comment by vmarcella** [comment]
@UPstartDeveloper Definitely. I'd look into chartist.js to see if there is a native css class that provides the aspect ratio needed. If you wanted to go above and beyond, you could also change the aspect ratio based upon the amount of data that was being rendered (Since 4:3 works really well for displaying less data and 16:9 works really well for displaying a lot of it.)

This may not entirely fix the bunching of data on the x axis, but should definitely improve the spacing.
